ipyleaflet/projections.py

```diff
@@ -48,7 +48,7 @@
         )
     ),
     EPSG5936=Bunch(
-        Basemap=dict(
+        ESRIBasemap=dict(
```

Comment:
Should it be `ESRIBasemap` or just `ESRI`?
We have `NASAGIBS`, without `Basemap`.

Reply:
Looks like the projection used for the ESRI base map is different from the one used for the layers. If that's the case then I think it's fine to have ESRIBasemap as the name for the projection, with a comment on "This projection is best suited for this layer(s) -> URL"

With NASA GIBS, they divide their products by projections so any 3031 layer should fit the same projections we are using for the base map.